sessions/recap-project-2/character-count-example/js/index.js

```diff
@@ -0,0 +1,15 @@
+console.clear();
+
+const questionElement = document.querySelector('[data-js="question"]');
+const amountLeft = document.querySelector('[data-js="amountLeft"]');
+const maxLength = questionElement.getAttribute('maxLength');
```

Comment:
The attribute is named `maxlength` with lowercase `l`

```suggestion
const maxLength = questionElement.getAttribute('maxlength');
```